index.js

```diff
@@ -353,7 +355,8 @@ Hyperlog.prototype.batch = function (docs, opts, cb) {
     })
 
     var added = nodes.length > 1 ? {} : null
-    var idx = 0
+    var seqIdx = 1
+    var changeIdx = 1
```

Comment:
We need to increment these separately. When a new node is deduplicated locally, its seq++ but its change will be the same as the existing doc. However, when a brand new node is inserted (no dedupe), its seq AND change must increment.